package Exception;

/**
 * Created by 12 on 11.04.2017.
 */
public class Test4 {
    public static void main(String[] args) {
        try {
            throw new Exception("first");
        } catch (Exception e){
            throw new RuntimeException(e);
        }
        Object
    }
}
